ios/Plugin/Plugin.swift

```diff
@@ -26,6 +26,10 @@ public class SecureStoragePlugin: CAPPlugin {
     @objc func get(_ call: CAPPluginCall) {
         let key = call.getString("key") ?? ""
         let hasValue = KeychainWrapper.standard.hasValue(forKey: key)
+        call.success([
+            "value": KeychainWrapper.standard.string(forKey: key) ?? ""
```

Comment:
if item with given key does not exist will it return null or empty string?

Reply:
It returns empty string, but i just changed it to return null.